app/views/catalog/_home_text.html.erb

```diff
@@ -1,5 +1,5 @@
 <div id="getting-started">
-  <h3>Welcome to Argo!</h3>
+  <h1 class="h3">Welcome to Argo!</h1>
```

Comment:
Fixes an a11y ding.

Reply:
Can I say how much I love the BS heading classes?

Reply:
@justinlittman 💯